torch/quantization/convert_modules.py

```diff
@@ -0,0 +1,222 @@
+from __future__ import absolute_import, division, print_function, unicode_literals
+import torch.nn as nn
+import torch.nn.quantized as nnq
+import torch
+from .observer import *
+import heapq
+
+def _forward_hook(self, input, output):
+    for module in self.children():
+        module.forward(output)
+
+def add_observer(module, qconfig_dict, qconfig_parent=None, prefix=''):
```

Comment:
the great debate - add_observer or add_observers? :)

Reply:
I'll use `add_observer`, we'll have `add_quant_dequant` as well